scipy/signal/filter_design.py

```diff
@@ -3692,11 +3692,10 @@ def cheb1ord(wp, ws, gpass, gstop, analog=False, fs=None):
         wp = 2*wp/fs
         ws = 2*ws/fs
 
-    filter_type = 2 * (len(wp) - 1)
-    if wp[0] < ws[0]:
+    # 1 = lp, 2 = hp, 3 = bs, 4 = bp
+    filter_type = 2 * len(wp) - 1
```

Comment:
This "simplification" looks a bit less obvious to me?

Reply:
I made it consistent with the other one, which is also simpler.  

len(wp) is either 1 or 2 (or invalid)

filter_type = 2 * (len(wp) - 1)
is basically {1: 0, 2: 2}
then
if wp<ws it becomes {1: 1, 2: 3}
if wp>ws it becomes {1: 2, 2: 4}

filter_type = 2 * len(wp) - 1
is basically {1: 1, 2: 3}
which is already correct for wp\<ws 
then
if wp>ws it becomes {1: 2, 2: 4}